liquibase-service/src/main/resources/db/rdb_modern/routines/022-sp_nrt_case_management_postprocessing-001.sql

```diff
@@ -0,0 +1,472 @@
+               nrt.adi_complexion,
+               nrt.adi_ehars_id,
+               nrt.adi_hair,
+               nrt.adi_height,
+               nrt.adi_height_legacy_case,
+               nrt.adi_other_identifying_info,
+               nrt.adi_size_build,
+               nrt.ca_init_intvwr_assgn_dt,
+               nrt.ca_interviewer_assign_dt,
+               nrt.ca_patient_intv_status,
+               nrt.case_oid,
+               nrt.case_review_status,
+               nrt.case_review_status_date,
+               nrt.cc_closed_dt,
+               dim.D_CASE_MANAGEMENT_KEY,
+               nrt.epi_link_id,
+               nrt.field_foll_up_ooj_outcome,
+               nrt.fl_fup_actual_ref_type,
+               nrt.fl_fup_dispo_dt,
+               nrt.fl_fup_disposition_cd,
+               nrt.fl_fup_disposition_desc,
+               nrt.fl_fup_exam_dt,
+               nrt.fl_fup_expected_dt,
+               nrt.fl_fup_expected_in_ind,
+               nrt.fl_fup_field_record_num,
+               nrt.fl_fup_init_assgn_dt,
+               nrt.fl_fup_internet_outcome,
+               nrt.fl_fup_internet_outcome_cd,
+               nrt.fl_fup_investigator_assgn_dt,
+               nrt.fl_fup_notification_plan_cd,
+               nrt.fl_fup_ooj_outcome,
+               nrt.fl_fup_prov_diagnosis,
+               nrt.fl_fup_prov_exm_reason,
+               nrt.fld_foll_up_expected_in,
+               nrt.fld_foll_up_notification_plan,
+               nrt.fld_foll_up_prov_diagnosis,
+               nrt.fld_foll_up_prov_exm_reason,
+               nrt.init_fup_clinic_code,
+               nrt.init_fup_closed_dt,
+               nrt.init_fup_initial_foll_up,
+               nrt.init_fup_initial_foll_up_cd,
+               nrt.init_fup_internet_foll_up_cd,
+               LEFT(nrt.init_foll_up_notifiable, 27) AS init_foll_up_notifiable,
+               nrt.init_fup_notifiable_cd,
+               LEFT(nrt.initiating_agncy, 20) AS initiating_agncy,
+               nrt.internet_foll_up,
+               inv.INVESTIGATION_KEY,
+               LEFT(nrt.ooj_agency, 20) AS ooj_agency,
+               nrt.ooj_due_date,
+               nrt.ooj_initg_agncy_outc_due_date,
+               nrt.ooj_initg_agncy_outc_snt_date,
+               nrt.ooj_initg_agncy_recd_date,
+               nrt.ooj_number,
+               nrt.pat_intv_status_cd,
+               nrt.status_900,
+               nrt.surv_closed_dt,
+               nrt.surv_investigator_assgn_dt,
+               nrt.surv_patient_foll_up,
+               nrt.surv_patient_foll_up_cd,
+               nrt.surv_prov_exm_reason,
+               nrt.surv_provider_contact,
+               nrt.surv_provider_contact_cd,
+               nrt.surv_provider_diagnosis,
+               nrt.surv_provider_exam_reason
+        INTO #temp_cm_table
+        FROM dbo.nrt_investigation_case_management nrt WITH (NOLOCK)
+            INNER JOIN dbo.INVESTIGATION inv WITH (NOLOCK) ON inv.CASE_UID = nrt.public_health_case_uid
```

Comment:
Baseline assumption, Investigation will always run first, correct?

Reply:
Yes, it's correct